library/proc_macro/src/lib.rs

```diff
@@ -191,6 +191,14 @@ impl ToString for TokenStream {
 /// Prints the token stream as a string that is supposed to be losslessly convertible back
 /// into the same token stream (modulo spans), except for possibly `TokenTree::Group`s
 /// with `Delimiter::None` delimiters and negative numeric literals.
+///
+/// Note: the exact form of the output is subject to change, e.g. there might
```

Comment:
You could consider wrapping the paragraphs inside `<div class="warning"></div>` (with an empty line between the tags and the content) and remove the leading `Note: ` to make rustdoc render a [warning block](https://doc.rust-lang.org/nightly/rustdoc/how-to-write-documentation.html?highlight=wa#adding-a-warning-block). This might be too much though.

Reply:
I was wondering if there was a better way to mark this paragraph, but I don't see any other use of this class in the library docs. But there are plenty of existing "Note:" examples. So I think I will leave it unchanged.

Reply:
Right, that's because warning blocks were added pretty recently and they haven't found much adoption yet. There's one use of them in the rustc docs since I encouraged Nils to make use of them once.

However, I see your point, do however you prefer.